docs/tutorials/broadcast-join.rst

```diff
@@ -0,0 +1,298 @@
+GeoMesa Spark: Aggregating Data
+===============================
+
+This tutorial will show you how to:
+
+1. Use GeoMesa with `Apache Spark <http://spark.apache.org/>`__ in Scala.
+2. Create and use DataFrames with our geospatial User Defined Functions.
+3. Calculate aggregate statistics using a covering set of polygons.
+4. Create a new simple feature type to represent this aggregation.
+5. Visualize the result as a choropleth.
```

Comment:
Hmm... I am perhaps too informal.  It appears that most people say "choropleth map".